x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/aggregation/blockhash/DoubleBlockHash.java

```diff
@@ -61,58 +65,61 @@ public void add(Page page, GroupingAggregatorFunction.AddInput addInput) {
 
     private LongVector add(DoubleVector vector) {
         long[] groups = new long[vector.getPositionCount()];
+        // TODO use the array flavored add
         for (int i = 0; i < vector.getPositionCount(); i++) {
-            groups[i] = hashOrdToGroupNullReserved(longHash.add(Double.doubleToLongBits(vector.getDouble(i))));
+            groups[i] = ordinator.add(Double.doubleToLongBits(vector.getDouble(i)));
         }
         return new LongArrayVector(groups, groups.length);
```

Comment:
We need to use an array style add to make this fast. It should be possible, but I'd like to wait for this in a follow up too.